lib/decompress/zstd_decompress.c

```diff
@@ -2557,8 +2557,12 @@ size_t ZSTD_decompressStream(ZSTD_DStream* zds, ZSTD_outBuffer* output, ZSTD_inB
             {   size_t const neededInSize = ZSTD_nextSrcSizeToDecompress(zds);
                 size_t const toLoad = neededInSize - zds->inPos;   /* should always be <= remaining space within inBuff */
                 size_t loadedSize;
-                if (toLoad > zds->inBuffSize - zds->inPos) return ERROR(corruption_detected);   /* should never happen */
-                loadedSize = ZSTD_limitCopy(zds->inBuff + zds->inPos, toLoad, ip, iend-ip);
+                if (ZSTD_isSkipFrame(zds)) {
```

Comment:
very minor : 
we have `int const isSkipFrame = ZSTD_isSkipFrame(zds);` a few lines below.
Maybe it could be upped here.
The idea is to use the same `isSkipFrame` value.
This is a guarantee that it remains the same value between the 2 places where `ZSTD_isSkipFrame(zds)` is currently invoked.